server/routes/actions/actions_routes.py

```
@@ -149,17 +150,32 @@ def list_actions(user_id):
+                # Ensure system actions exist for this org
+                cur.execute("SELECT org_id FROM users WHERE id = %s", (user_id,))
+                org_row = cur.fetchone()
+                if org_row and org_row[0]:
+                    cur.execute(
+                        "SELECT 1 FROM actions WHERE org_id = %s AND is_system = true LIMIT 1",
+                        (org_row[0],),
+                    )
+                    if not cur.fetchone():
+                        try:
+                            seed_system_actions(org_row[0], user_id)
+                        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)